apps/web/src/services/administration/constants/user-table-constant.ts

```diff
@@ -80,7 +82,7 @@ export const userSearchHandlers = {
             label: 'Tags',
             dataType: 'object',
         }],
-    }],
+    }] as KeyItemSet[],
```

Comment:
suggestion: Type Annotation is better than Type Assertion. ([link](https://baek.dev/til/typescript/effective-typescript/ch02/item9))